configs/RCCS-Fugaku-Fujitsu-A64FX-TofuD/auxiliary_software_files/packages.yaml

```diff
@@ -5,42 +5,47 @@
 
 packages:
   all:
-    compiler: [fj, clang, gcc]
+    compiler: [clang, fj, gcc]
     providers:
       mpi: [fujitsu-mpi, openmpi, mpich]
       blas: [fujitsu-ssl2, openblas]
       lapack: [fujitsu-ssl2, openblas]
       scalapack: [fujitsu-ssl2, netlib-scalapack]
-      fftw-api: [fujitsu-ssl2, fftw, rist-fftw]
+      fftw-api: [fujitsu-fftw, fftw, rist-fftw]
     permissions:
       write: group
   htslib:
     version: [1.12]
   python:
     externals:
-      - spec: "python@3.10.8%fj@4.8.1 +ssl+tkinter arch=linux-rhel8-a64fx"
-        prefix: /vol0004/apps/oss/spack-v0.19/opt/spack/linux-rhel8-a64fx/fj-4.8.1/python-3.10.8-5q3ncyl2my7oomopsmukduqo36u6pnkg
+      - spec: "python@3.11.6%fj@4.10.0 arch=linux-rhel8-a64fx"
+        prefix: /vol0004/apps/oss/spack-v0.21/opt/spack/linux-rhel8-a64fx/fj-4.10.0/python-3.11.6-qbmpmn2uxu4oe3qoawxbizp7awqlgkcq
   openssh:
     permissions:
       write: user
   fujitsu-mpi:
+    buildable: False
     externals:
-      - spec: "fujitsu-mpi@4.8.1%fj@4.8.1 arch=linux-rhel8-a64fx"
-        prefix: /opt/FJSVxtclanga/tcsds-mpi-1.2.36
-      - spec: "fujitsu-mpi@4.8.1%clang@15.0.3 arch=linux-rhel8-a64fx"
-        prefix: /opt/FJSVxtclanga/tcsds-mpi-1.2.36
-      - spec: "fujitsu-mpi@4.8.1%gcc@12.2.0 arch=linux-rhel8-a64fx"
+      - spec: "fujitsu-mpi@4.10.0%clang@17.0.2 arch=linux-rhel8-a64fx"
+        prefix: /opt/FJSVxtclanga/tcsds-mpi-1.2.38
+      - spec: "fujitsu-mpi@4.10.0%fj@4.10.0 arch=linux-rhel8-a64fx"
+        prefix: /opt/FJSVxtclanga/tcsds-mpi-1.2.38
+      - spec: "fujitsu-mpi@4.10.0%gcc@13.2.0 arch=linux-rhel8-a64fx"
         prefix: /vol0004/apps/oss/mpigcc/fjmpi-gcc12
-    buildable: False
   fujitsu-ssl2:
+    buildable: False
     externals:
-      - spec: "fujitsu-ssl2@4.8.1%fj@4.8.1 arch=linux-rhel8-a64fx"
-        prefix: /opt/FJSVxtclanga/tcsds-ssl2-1.2.36
-      - spec: "fujitsu-ssl2@4.8.1%clang@15.0.3 arch=linux-rhel8-a64fx"
-        prefix: /opt/FJSVxtclanga/tcsds-ssl2-1.2.36
-      - spec: "fujitsu-ssl2@4.8.1%gcc@12.2.0 arch=linux-rhel8-a64fx"
-        prefix: /opt/FJSVxtclanga/tcsds-ssl2-1.2.36
+      - spec: "fujitsu-ssl2@4.10.0%clang@17.0.2 arch=linux-rhel8-a64fx"
+        prefix: /vol0004/apps/oss/llvm-v17.0.2/compute_node
+      - spec: "fujitsu-ssl2@4.10.0%fj@4.10.0 arch=linux-rhel8-a64fx"
+        prefix: /opt/FJSVxtclanga/tcsds-ssl2-1.2.38
+      - spec: "fujitsu-ssl2@4.10.0%gcc@13.2.0 arch=linux-rhel8-a64fx"
+        prefix: /opt/FJSVxtclanga/tcsds-ssl2-1.2.38
+  rist-fftw:
```

Comment:
(minor) I'm not aware of a `rist-fftw` package (or virtual), at least in the builtin repo

Reply:
it's an external package provided and maintained by Research Organization for Information Science and Technology (RIST)